src/calliope/example_models/national_scale/scenarios.yaml

```diff
@@ -207,3 +207,52 @@ overrides:
   capacity_factor:
     techs.ccgt.capacity_factor_min: 0.8
     techs.ccgt.capacity_factor_max: 0.9
+
+  load_data_sources:
```

Comment:
I think for an example model, it would be clearer to create an entirely separate copy of the model where we showcase how the read-from-file functionality mixed together with old-style YAML leads to clearer model formulation.